src/expression/types/projection_definition.ts

```diff
@@ -0,0 +1,36 @@
+export class ProjectionDefinition {
+    public from
```

Comment:
You can add `public` modifier before every field in the constructor to save the need to write this.
But if you don't please specify the type here.

Reply:
I've fixed that too now.